src/llamafactory/chat/hf_engine.py

```diff
@@ -177,14 +228,25 @@ def _chat(
         processor: Optional["ProcessorMixin"],
         template: "Template",
         generating_args: Dict[str, Any],
+        model_args: "ModelArguments",
         messages: Sequence[Dict[str, str]],
         system: Optional[str] = None,
         tools: Optional[str] = None,
         image: Optional["NDArray"] = None,
         input_kwargs: Optional[Dict[str, Any]] = {},
     ) -> List["Response"]:
         gen_kwargs, prompt_length = HuggingfaceEngine._process_args(
-            model, tokenizer, processor, template, generating_args, messages, system, tools, image, input_kwargs
+            model,
```

Comment:
This line can be collapsed to one line
other modify like this should be processed more carefully

Reply:
Thx I've learnt a lots today.

Reply:
I suggest you don't use pycharm/vscode auto formatter to format code, as they might bring more diffs
you could directly use `make` to format code

Reply:
Okay!!